### ISSUE TITLE
Where are the coverage reports

### ISSUE THREAD

**Issue by alex-konovalov** [semigroups/Semigroups#844]
https://app.codecov.io/gh/semigroups/Semigroups/ shows code coverage checks - but why they are not reported as comments in pull requests? Maybe @fingolfin knows?

**Comment by wilfwilson** [comment]
Because such comments are turned off in the `.codecov.yml` file. They're easy enough to turn on again.

**Comment by james-d-mitchell** [comment]
They seem to be back!

**Comment by james-d-mitchell** [comment]
I'd prefer that PR's not fail because of insufficient coverage because it's a bit flaky, if that's also possible?

**Comment by fingolfin** [comment]
Yes; you can also set a threshold, so that a failure is only declared if the coverage falls more than that threshold. See https://docs.codecov.com/docs/common-recipe-list which even has examples for just this case
